dockers/docker-database/database_config.json.j2

```diff
@@ -121,6 +121,12 @@
             "instance" : "redis"
         }
 {% endif %}
+        ,
+        "EVENT_DB" : {
+            "id" : 19,
+            "separator": "|",
+            "instance" : "redis"
+        }
```

Comment:
This a new redis DB instance addition. Have few queries:

1. is there a max limit of redis DB instances that can be spawned and maintained on a given SONiC instance?
2. If so, was it ensured that its within limits and no impact to the overall system bring-up and working in steady state?
3. Is this a global database i.e. only a single instance of it would be spawned on the system in global namespace? or would it be one per ASIC/NPU namespace? Where is that determined/specified in this changeset?